ingest/ledgerbackend/stellar_core_runner_test.go

```diff
@@ -12,6 +12,12 @@ import (
 	"github.com/stellar/go/support/log"
 )
 
+func newUint(v uint) *uint {
+	p := new(uint)
+	*p = v
+	return p
+}
```

Comment:
Nit, `v` is a copy so I think below will work too:
```suggestion
func newUint(v uint) *uint {
	return &v
}
```